app/src/main/java/com/keylesspalace/tusky/fragment/TimelineFragment.java

```diff
@@ -1055,11 +1056,8 @@ private void updateAdapter() {
         @Override
         public void onInserted(int position, int count) {
             adapter.notifyItemRangeInserted(position, count);
-            if (position == 0
-                    && layoutManager.findFirstVisibleItemPosition() == 0
-                    && (swipeRefreshLayout.getVisibility() == View.VISIBLE
-                    || progressBar.getVisibility() == View.VISIBLE)) {
-                recyclerView.post(() -> layoutManager.scrollToPosition(0));
+            if (position == 0) {
+                recyclerView.scrollBy(0, Utils.dpToPx(Objects.requireNonNull(getContext()), -30));
```

Comment:
you could just use requireContext() instead of wrapping it into requireNonNull

Reply:
Sure thing